mail_message.go

```diff
@@ -351,6 +357,24 @@ func (message *MailMessage) postToBlog(senderUser *User) error {
 	return nil
 }
 
+// Sends email data to a Mattermost webhook to post on the town-square channel
+func (message *MailMessage) postToMattermost() error  {
+	payload := fmt.Sprintf("payload = {\"channel\": \"town-square\", \"username\": \"Team 254 Mailing List Bot\", \"icon_url\": \"http://media.team254.com/2016/10/8fdb07a0-254-Swoosh.png\", \"text\": \"<!channel>\" %s}", message.body.HTML)
```

Comment:
Use `json.Marshal` and an inline struct to do this instead of hardcoding JSON. See https://github.com/Team254/cheesy-arena/blob/master/web/api.go#L148 for an example.